FileReader.py

```diff
@@ -1,12 +1,8 @@
 import os
 
-class FileReader:
-    def __init__(self, file_path : str) -> None:
-        if not os.path.isfile(file_path):
-            raise Exception(f"File {file_path} could not be found.")
-        self.__file_path = file_path
-    
-    def read_lines(self) -> str:
-        with open(self.__file_path, mode="r") as file:
-            content = file.readlines()
-        return content
\ No newline at end of file
+def read_lines(file_path) -> str:
```

Comment:
please rename the module to `file_reader.py` (python standards)
and add a type to the param